cpp/oneapi/dal/algo/pca/train_types.hpp

```diff
@@ -68,16 +69,22 @@ class ONEAPI_DAL_EXPORT train_result {
         return *this;
     }
 
-    auto& set_explained_variance(const table& value) {
-        set_explained_variance_impl(value);
+    auto& set_variances(const table& value) {
```

Comment:
Shouldn't it be marked as `inline`?

Reply:
No, this is class member function, it's `inline` by design